lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -336,6 +335,7 @@ def add_block(xblock):
             blocks[unicode(xblock.location)] = {
                 u'id': unicode(xblock.location),
                 u'type': xblock.category,
+                u'parent': None,
```

Comment:
In these tests parent will always be set to `None` by api and we verify that the value of parent is indeed set to None.